src/SmartComponents/Systems/SystemsTable.js

```diff
@@ -143,6 +159,9 @@ const SystemsTable = ({
             columns={(defaultColumns) => systemsColumnsMerger(defaultColumns, systemsListColumns)}
             showTags
             customFilters={{
+                ...operatingSystemFilter ? {
+                    filters: [...(osFilter || [])]
+                } : {},
```

Comment:
I think this is something we could cover with tests: there is SystemTable.test.js available where we could add a scenario where query parameters contain `filter[os]`. And given that InventoryTable receives customFilters, we can assert on what the mocked component was called with and make sure osFilter is passed in the expected format. WDYT?